spec.go

```diff
@@ -0,0 +1,15 @@
+package clustertpr
+
+import "github.com/giantswarm/clustertpr/spec"
+
+type Spec struct {
```

Comment:
Renamed top level struct to `Spec`. It was `Cluster` before, but is also have a field named `Cluster`.